### PR TITLE
docs: doc nit fixes, broken links, missing pages, etc.

### PR DESCRIPTION
**Pull request by denniswon**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
This PR updates the documentation by adding backticks to `Promise` instances and fixing typos related to entrypoints.

### Detailed summary
- Added backticks to `Promise` instances for consistency
- Corrected typos from "Entrypoint" to "EntryPoint" in multiple files

> The following files were skipped due to too many changes: `site/packages/aa-signers/portal/authenticate.md`, `site/packages/aa-accounts/light-account/signMessageWith6492.md`, `site/packages/aa-signers/portal/getAuthDetails.md`, `site/packages/aa-signers/fireblocks/getAuthDetails.md`, `site/packages/aa-accounts/light-account/signTypedDataWith6492.md`, `site/packages/aa-core/smart-account-client/actions/signMessageWith6492.md`, `site/packages/aa-core/smart-account-client/actions/checkGasSponsorshipEligibility.md`, `site/packages/aa-core/smart-account-client/actions/signTypedDataWith6492.md`, `site/packages/aa-signers/passport/getAuthDetails.md`, `site/packages/aa-signers/capsule/authenticate.md`, `packages/core/e2e-tests/simple-account-staging.e2e.test.ts`, `site/packages/aa-signers/capsule/getAuthDetails.md`, `site/packages/aa-accounts/light-account/actions/transferOwnership.md`, `site/packages/aa-signers/web3auth/authenticate.md`, `site/packages/aa-signers/arcana-auth/authenticate.md`, `site/packages/aa-signers/web3auth/getAuthDetails.md`, `site/packages/aa-signers/arcana-auth/getAuthDetails.md`, `site/packages/aa-signers/magic/authenticate.md`, `site/packages/aa-signers/magic/getAuthDetails.md`, `site/packages/aa-core/smart-account-client/actions/buildUserOperationFromTx.md`, `site/packages/aa-signers/turnkey/authenticate.md`, `site/packages/aa-signers/turnkey/getAuthDetails.md`, `site/packages/aa-signers/lit-protocol/authenticate.md`, `site/packages/aa-signers/lit-protocol/getAuthDetails.md`, `site/packages/aa-core/smart-account-client/actions/waitForUserOperationTransaction.md`, `site/packages/aa-core/smart-account-client/actions/sendUserOperation.md`, `site/packages/aa-signers/particle/getAuthDetails.md`, `site/packages/aa-signers/particle/authenticate.md`, `site/packages/aa-signers/passport/authenticate.md`, `site/packages/aa-core/smart-account-client/actions/signUserOperation.md`, `packages/core/src/client/decorators/bundlerClient.ts`, `site/packages/aa-core/smart-account-client/actions/dropAndReplaceUserOperation.md`, `site/packages/aa-core/bundler-client/index.md`, `site/resources/terms.md`, `packages/ethers/CHANGELOG.md`, `packages/plugingen/CHANGELOG.md`, `site/CHANGELOG.md`, `packages/alchemy/CHANGELOG.md`, `packages/core/CHANGELOG.md`, `packages/accounts/CHANGELOG.md`, `CHANGELOG.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/core/src/client/decorators/bundlerClient.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import type {
   Transport,
 } from "viem";
 import { estimateUserOperationGas } from "../../actions/bundler/estimateUserOperationGas.js";
-import { getSupportedEntryPoints } from "../../actions/bundler/getSupportedEntrypoints.js";
+import { getSupportedEntryPoints } from "../../actions/bundler/getSupportedEntryPoints.js";
```
Comment (avasisht23) [inline packages/core/src/client/decorators/bundlerClient.ts:11]: did you mean to change this?

Comment (avasisht23) [reply packages/core/src/client/decorators/bundlerClient.ts:11]: i think the builds fails on this pr bc of this

Comment (denniswon) [reply packages/core/src/client/decorators/bundlerClient.ts:11]: huh git did not pick up the file renamed. fixed and pushed again.

**Review by avasisht23 — DISMISSED** [review]

File: site/packages/aa-accounts/light-account/actions/transferOwnership.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ const transferredClient = await createLightAccountClient({
 
 ### `Promise<0x${string}>`
 
-A Promise containing the hash of either the UO or transaction containing the UO which transferred ownership of the smart account's owner.
+A `Promise` containing the hash of either the UO or transaction containing the UO which transferred ownership of the smart account's owner.
```
Comment (dphilipson) [inline site/packages/aa-accounts/light-account/actions/transferOwnership.md:53]: (Doesn't need to be changed in this PR) Wait, how does this API work? When does it return the UO hash and when the transaction hash? Those are two very different things and feels like it needs clarity.

File: site/packages/aa-core/accounts/index.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ export type SmartContractAccount<Name extends string = string> =
     getInitCode: () => Promise<Hex>;
     isAccountDeployed: () => Promise<boolean>;
     getFactoryAddress: () => Address;
-    getEntrypoint: () => Address;
+    getEntryPoint: () => Address;
```
Comment (dphilipson) [inline site/packages/aa-core/accounts/index.md:86]: (Not specifically related to 0.7, no change required) I feel like the SDK can be a little inconsistent about when to have a field vs a getter. Seems like at some point when we do some cleanup, we might consider having a convention that we only use getters when it returns a promise, but for plain values just a field e.g. `factoryAddress: Address`.

**Review by dphilipson — APPROVED** [review]
Thanks for this cleanup!

**Pull request merged**